src/sentry/features/temporary.py

```diff
@@ -450,6 +450,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:insights-query-date-range-limit", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=True)
     # Make Insights modules use EAP instead of metrics
     manager.add("organizations:insights-use-eap", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    # Make Insights modules use EAP instead of metrics, but for area's that are still WIP
+    manager.add("organizations:insights-use-eap-wip", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
The existing `"organizations:insights-use-eap"` flag name really painted you into a corner for this second flag, eh? Honestly not a bad idea to replace that flag with `"organizations:insights-llm-module-use-eap"` but not important.

Anyway, I think `"organization:insights-all-modules-use-eap"` is a little better than the `"-wip"` prefix, since a feature flag is kind of a WIP by nature, but again, not that important.

Reply:
Thinking about this a bit more, why not use the existing feature flag? Any functionality that doesn't work on EAP won't work until it's implemented, which makes sense, right?